packages/schema-files-loader/src/usesPrismaSchemaFolder.ts

```diff
@@ -1,5 +1,9 @@
 // Minimal subset of the return type of `getConfig` from `@prisma/internals`, used for `usesPrismaSchemaFolder`.
-// This is a simplified version of the actual `ConfigMetaFormat` type, which isn't imported to avoid circular dependencies.
+// This is a simplified version of the actual `ConfigMetaFormat` type, which isn't imported to avoid circular dependencies.export
```

Comment:
```suggestion
// This is a simplified version of the actual `GetConfigResponse` type, which isn't imported to avoid circular dependencies.export
```